src/Domain/Model/Throttle.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2018 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Domain\Model;
+
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Throttle extends Eloquent
+{
+    protected $table = 'throttle';
+
+    public function user(): BelongsTo
+    {
+        return $this->belongsTo(User::class, 'user_id');
+    }
+
+    public function setUpdatedAt($value)
+    {
+        /**
+         * This is the dirty way to tell Illuminate that we don't have an updated at field
+         * while still having a created_at field.
```

Comment:
Ha, but we don't have a `created_at` field, do we?

![b3qijlfccaejlwm jpg-large](https://user-images.githubusercontent.com/605483/40573781-c96cb35a-60c6-11e8-98d6-476add20b1e3.jpeg)

https://twitter.com/nzkoz/status/538892801941848064

😆